### DISCUSSION TITLE
Anonymous DevTools function

### DISCUSSION THREAD

**Discussion by motionrus** [pmndrs/zustand#2660] [Q&A]
I don't want to write the name every time, is there a way to automatically put the name for an anonymous function
<img width="485" alt="image" src="https://github.com/user-attachments/assets/1b58209a-5cc6-4ad0-8a55-593aa9161690">

**Comment by dai-shi** [comment]
Have you tried this?
https://github.com/pmndrs/zustand/blob/66f3a029fbc4640b76c26959e01a5caa857c04dc/readme.md?plain=1#L422-L426

**Reply by fazlulShanto** [reply]
did you find a working solution?

**Comment by Poyoman39** [comment]
This https://github.com/pmndrs/zustand/pull/3250 may solve
